app/assets/javascripts/components/dashboard/edit_task.jsx

```diff
@@ -27,6 +27,19 @@ class TaskEditForm extends DefaultModal {
     });
   }
 
+  componentWillReceiveProps(nextProps) {
+    // Check if props are equivalent???
```

Comment:
delete??

Reply:
you should have a check here that goes like

```javascript
if (this.props != nextProps) {
  
}
```

and use it to wrap around your requester get